src/Workspaces/CSharp/Portable/Extensions/ITypeSymbolExtensions.TypeSyntaxGeneratorVisitor.cs

```diff
@@ -145,6 +147,21 @@ public TypeSyntax CreateSimpleTypeSyntax(INamedTypeSymbol symbol)
                     SyntaxFactory.TypeArgumentList(SyntaxFactory.SeparatedList(typeArguments)));
             }
 
+            private TupleTypeSyntax CreateTupleTypeSyntax(ITupleTypeSymbol symbol)
+            {
+                var list = new SeparatedSyntaxList<TupleElementSyntax>();
+                ImmutableArray<ITypeSymbol> types = symbol.Types;
```

Comment:
Sure, I'll change.
FYI, on the compiler side, the guideline I heard were to spell out the type if it doesn't appear elsewhere on the line (explicit cast, constructor, etc).